src/main/java/com/example/provider/json/SerializedTrip.java

```diff
@@ -33,6 +34,8 @@ abstract class SerializedTrip {
   abstract ImmutableList<Waypoint> waypoints();
   /** ID of the vehicle. */
   abstract String vehicleId();
+  /** Vehicle route list. */
+  abstract List<LatLng> routeList();
```

Comment:
Let's use `SerializedLatLng` here so the client can use the fields "latitude" and "longitude" without the "_" suffix.